src/plugins/intel_gpu/include/intel_gpu/primitives/softmax.hpp

```diff
@@ -25,24 +25,13 @@ namespace cldnn {
 struct softmax : public primitive_base<softmax> {
     CLDNN_DECLARE_PRIMITIVE(softmax)
 
-    /// @brief Enum type to specify softmax's normalization scope (see #dimension).
-    enum dimension_t {
-        normalize_b,
-        normalize_f,
-        normalize_x,
-        normalize_y,
-        normalize_z,
-        normalize_fyx,
-        normalize_all
-    };
-
     /// @brief Constructs softmax primitive.
     /// @param id This primitive id.
     /// @param input Input primitive id.
-    /// @param dimension Defines a scope of normalization (see #dimension).
+    /// @param dimension Defines a scope of normalization
     softmax(const primitive_id& id,
             const primitive_id& input,
-            const dimension_t dimension = normalize_fyx,
+            const int64_t dimension = 1,
```

Comment:
let's remove default value for dimension

Reply:
Default value matches operation spec